src/js/App/Sidenav/Navigation.js

```diff
@@ -12,172 +12,126 @@ import './Navigation.scss';
 
 const basepath = document.baseURI;
 
-const openshiftLinks = {
-    supportcases: {
+const extraLinks = {
```

Comment:
Can we make sure the Documentation link is always the last one?

Reply:
Sure thing! I've made it to match what was here previously, but I've seen the ticket about matching position of all docs link everywhere.